.tekton/doc.yaml

```diff
@@ -33,7 +33,7 @@ spec:
                 resolver: http
                 params:
                   - name: url
-                    value: https://raw.githubusercontent.com/tektoncd/catalog/main/stepaction/git-clone/0.1/git-clone.yaml
+                    value: https://raw.githubusercontent.com/tektoncd-catalog/git-clone/af93731eb19a28e13f967b7cee8269d1f75dfe6b/task/git-clone/git-clone.yaml
```

Comment:
@chmouel this is the one https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/refs/heads/p/tasks/task-git-clone/0.4.1/task-git-clone.yaml which we ship with Operator can we use this ??